### PR TITLE
homework 2 is done

### PR DESCRIPTION
**Pull request by 821125**

### CODE REVIEW

File: homework_2/HW-2-1.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    'Название ОС',
+    'Код продукта',
+    'Тип системы'
+]]
+
+def get_data():
+    data = []
+    dirctory_items = list(filter(lambda x: '.txt' in x, os.listdir()))
+    for filename in dirctory_items:
+        with open(filename) as file:
```
Comment (ibarbylev) [inline homework_2/HW-2-1.py:18]: Unfortunately, this method will throw an error for Linux and macOS users. 
(Before opening an unknown file, you must detect its encoding)

File: homework_2/HW-2-1.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        os_code_list.append(item[1]) if item[0] == main_data[0][2] else None
+        os_type_list.append(item[1]) if item[0] == main_data[0][3] else None
+    for i in range(len(os_prod_list)):
+        main_data.append([os_prod_list[i], os_name_list[i], os_code_list[i], os_type_list[i]])
+    return main_data
+
+def write_to_csv(file_path):
+    data = get_data()
+    with open(file_path, 'w', encoding='utf-8', newline='') as csv_file:
+        writer = csv.writer(csv_file, delimiter=',', quoting=csv.QUOTE_NONNUMERIC)
```
Comment (ibarbylev) [inline homework_2/HW-2-1.py:33]: What's the point of adding ```newline=''``` and ```delimiter=',', quoting=csv.QUOTE_NONNUMERIC'''?

Comment (821125) [reply homework_2/HW-2-1.py]: Newline='' was added for remove extra line, other options were removed in the last commit.

File: homework_2/HW-2-1.py (hunk 1 above)
Comment (821125) [reply homework_2/HW-2-1.py]: Fixed with function.

**Pull request closed without merging**